firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionImpl.java

```diff
@@ -306,12 +310,17 @@ private UpdateTask updateApp(boolean showDownloadInNotificationManager) {
   }
 
   @Override
-  public void startFeedback() {
-    startFeedback(Executors.newSingleThreadExecutor());
+  public void startFeedback(int infoTextResourceId) {
+    startFeedbackWithInfoText(firebaseApp.getApplicationContext().getText(infoTextResourceId));
```

Comment:
Can you chain to `startFeedback(String)` instead of directly to `startFeedbackWithInfoText`, so the implementations are less likely to diverge?

Reply:
Actually I'm realizing our API should have `startFeedback(CharSequence)`, so yes.

I was trying to model our API after [TextView.setText](https://developer.android.com/reference/android/widget/TextView#setText(java.lang.CharSequence)), which accepts an int or a CharSequence. I thought it was int or String.